libs/core/langchain_core/indexes/__init__.py

```diff
@@ -0,0 +1,4 @@
+from langchain_core.indexes.base import Index
```

Comment:
@baskaryan we have an `indexing` namespace in langchain-core. Let's keep in the same namespace, otherwise, it's going to be confusing?